client/client.go

```diff
@@ -595,6 +595,12 @@ type TargetSignedStruct struct {
 	Signatures []data.Signature
 }
 
+type ErrNoSuchTarget string
```

Comment:
There seems to be a lint error because this error doesn't have a comment.  (You can verify with `make lint`).